static/js/src/advantage/subscribe/react/components/ModalSteps/StepOne.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { ActionButton } from "@canonical/react-components";
+import { useFormikContext } from "formik";
+
+import useStripeCustomerInfo from "../../APICalls/useStripeCustomerInfo";
+import PaymentMethodForm from "../PaymentMethodForm";
+import useProduct from "../../APICalls/useProduct";
+import usePreview from "../../APICalls/usePreview";
+import ModalHeader from "../ModalParts/ModalHeader";
+import ModalBody from "../ModalParts/ModalBody";
+import ModalFooter from "../ModalParts/ModalFooter";
+
+type StepOneProps = {
+  error: React.ReactNode | null;
```

Comment:
`ReactNode` can be `null` be default
```suggestion
  error: React.ReactNode;
```
> type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;